src/einsteinpy/symbolic/christoffel.py

```diff
@@ -107,3 +107,23 @@ def change_config(self, newconfig="lll", metric=None):
             new_tensor, self.syms, config=newconfig, parent_metric=metric
         )
         return new_obj
+
+    def lorentz_transform(self, transformation_matrix):
+        """
+        Performs a Lorentz transform on the tensor.
+
+        Parameters
+        ----------
+            transformation_matrix : ~sympy.tensor.array.dense_ndim_array.ImmutableDenseNDimArray or list
+                Sympy Array or multi-dimensional list containing Sympy Expressions
+
+        Returns
+        -------
+            ~einsteinpy.symbolic.tensor.ChristoffelSymbols
```

Comment:
`~einsteinpy.symbolic.christoffel.ChristoffelSymbols`. Kindly take note that `ChristoffelSymbols` resides in `symbolic/christoffel.py`